src/postprocess/VrHooks.cpp

```diff
@@ -34,13 +35,88 @@ namespace {
 		return (T)hooksToOriginal[hookFunction];
 	}
 
+	BOOL IsBadMemPtr(BOOL write, void *ptr, size_t size)
```

Comment:
This is the part that doesn't sit right with me. There is no way the original `GetRecommendedRenderTargetSize` has this kind of protection against incorrect calls, and the fact that this is an issue for our hooked call hints at a deeper issue. Given that, even with this workaround, the support for Alyx is only half complete (due to missing support for dynamic resolution), I'd prefer not to put this into the code base at this time.

Reply:
I agree, this is a hacky workaround for an issue we don't understand. I'm not sure I'll be inclined to look deeper into this, but if I do and find out anything new I'll let you know.

Reply:
I'd be happy to merge the missing function definitions, though. If you want, you could open another PR with just those, minus the BadMemPtr hack :)

Reply:
See #128.